_tests/browser_tests/jenkins_conf.js

```diff
@@ -0,0 +1,41 @@
+var env = require('./environment.js');
+
+exports.config = {
+  framework: 'jasmine2',
+  specs: ['spec_suites/shared/*.js'],
+  capabilities: {
+    'browserName': 'chrome',
+    'name': 'flapjack-browser-tests'
+  },
+  
+  sauceUser: process.env.SAUCE_USER,
+  sauceKey: process.env.SAUCE_KEY,
+
+  onPrepare: function() {
+    return browser.ignoreSynchronization = true;
+
+
+    var jasmineReporters = require('jasmine-reporters');
+    var mkdirp = require('mkdirp');
+    var SpecReporter = require('jasmine-spec-reporter');
+
+    // add jasmine spec reporter
+    jasmine.getEnv().addReporter(new SpecReporter({displayStacktrace: true}));
+
+    // var folderName = (new Date()).toString().split(' ').splice(1, 4).join('-');
+    var newFolder = "reports/"
+
+    mkdirp(newFolder, function( err ) {
```

Comment:
Ideally move the anonymous function outside of the nesting and give it a name, so that the call can look like:

`mkdirp( newFolder, _processFolder );`